.travis/packages.txt

```diff
@@ -68,8 +68,7 @@ Nef_3
 Nef_S2 
 NewKernel_d 
 Number_types 
-OpenNL 
-Operations_on_polyhedra 
+OpenNL  
```

Comment:
There is one more trailing space, after `OpenNL `. Our tools may be stupid and report an error about it.